src/usr/share/gpgmailer/gpgkeyverifier.py

```diff
@@ -93,68 +94,55 @@ def get_expiration_warning_message(self, loop_current_time):
     #   with a receiver.
     #
     # config: The config dictionary read from the program configuration file.
-    def _initialize_email_dicts(self, config):
+    def _initialize_key_dicts(self, config):
 
-        recipient_fingerprints = []
+        # Build a dict of fingerprints from the keyring that includes associated email addresses,
+        # whether the key is the gpgmailer configured sender, and whether any expiry emails have
+        # been sent for this key.
+        fp_to_key_dict = self.gpgkeyring.fingerprint_to_key_dict
```

Comment:
I really don't like acronyms in variable names. However, I may be convinced to let it go. Thoughts?